app/views/list/list.rss.erb

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<rss version="2.0" xmlns:discourse="http://www.discourse.org/" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
+  <channel>
+    <% lang = SiteSetting.find_by_name('default_locale').try(:value) %>
```

Comment:
I just copied [`list.rss.erb`](https://github.com/discourse/discourse/blob/01341f33e3f16777767d0ad80008c52d4d44d941/app/views/list/list.rss.erb#L1-L37) maybe I could use the one from core?